components/batch-system/src/batch.rs

```diff
@@ -10,19 +10,24 @@ use crate::config::Config;
 use crate::fsm::{Fsm, FsmScheduler, Priority};
 use crate::mailbox::BasicMailbox;
 use crate::router::Router;
-use crossbeam::channel::{self, SendError};
+use crossbeam::channel::{self, Receiver, SendError, Sender};
 use file_system::{set_io_type, IOType};
+use lazy_static::lazy_static;
 use std::borrow::Cow;
-use std::sync::atomic::AtomicUsize;
-use std::sync::Arc;
-use std::thread::{self, JoinHandle};
+use std::sync::atomic::{AtomicUsize, Ordering};
+use std::sync::{Arc, Mutex};
+use std::thread::{self, current, JoinHandle, ThreadId};
 use std::time::Duration;
 use tikv_util::mpsc;
 use tikv_util::time::Instant;
 use tikv_util::{debug, error, info, safe_panic, thd_name, warn};
 
+lazy_static! {
+    pub static ref SCALE_FINISHED_CHANNEL: (Sender<()>, Receiver<()>) = channel::bounded(1);
```

Comment:
It looks too hack...

Reply:
> It looks too hack...

Yes, I agree. My requirement is to share a `channel` between the `config manager of raftstore` and `Pollers` to achieve synchronization, avoid that the last `Pollers` adjustment have not been completed when the next configuration update request comes. I thought about `callback` before, but can I get the same `channel` when I add `Pollers` multiple times to ensure that the `config manager` can be notified when I drop `Pollers`?

@NingLin-P   Would you please give me some suggestions?